### DISCUSSION TITLE
Highlight text in pdf

### DISCUSSION THREAD

**Discussion by saqlainch92786** [pdfme/pdfme#662] [Ideas]
Hey All , can we achieve highlighting the text in pdfme as a rich feature?

**Comment by shannon1102** [comment]
Add bold, and italic more

**Comment by jonelofficial** [comment]
I need this feature also: like highlighting specific word on a text

**Comment by hand-dot** [comment]
ref: https://github.com/pdfme/pdfme/issues/564
